static/app/views/performance/transactionSummary/transactionSpans/opsFilter.tsx

```diff
@@ -75,6 +75,8 @@ export default function OpsFilter(props: Props) {
           location={location}
           orgSlug={organization.slug}
           eventView={opsFilterEventView}
+          cursor="0:0:1"
```

Comment:
Another one? Hmmmmm there should be a cleaner approach than this. I think maybe we should have a way for queries to change which url param they (or the EventView payload function that gets called) looks at for the cursor, and use `null` to indicate that the `EventView` should not use the location cursor?

Reply:
Yeah, there should definitely be a clean way to handle this. Going to merge this for now and revisit later.